extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -238,12 +239,23 @@ export class ConsentUI {
    * @return {!Element}
    */
   createPlaceholder_() {
-    // TODO(@zhouyx): Allow publishers to provide placeholder upon request
     const placeholder = this.parent_.ownerDocument.createElement('placeholder');
     toggle(placeholder, false);
     const {classList} = placeholder;
-    classList.add(consentUiClasses.fill);
-    classList.add(consentUiClasses.placeholder);
+    classList.add(consentUiClasses.loadingPlaceholder);
+
+    const loadingSpinner = htmlFor(placeholder)`
+    <svg viewBox="0 0 40 40">
+      <defs>
+        <linearGradient id="grad">
+          <stop stop-color="rgb(105, 105, 105)"></stop>
+          <stop offset="100%"  stop-color="rgba(105, 105, 105, 0)"></stop>
```

Comment:
I had to update the code to make it work on Safari correctly. You'll need to do:

 `stop-color="rgb(105, 105, 105) stop-opacity="0"`

instead here

Reply:
Thanks! Will update now 😄